src/autocluster_k8s.erl

```diff
@@ -67,22 +67,34 @@ make_request() ->
       [{"Authorization", ["Bearer ", Token1]}],
       [{ssl, [{cacertfile, autocluster_config:get(k8s_cert_path)}]}]).
 
-%% @spec node_name(k8s_endpoint) -> list()
+%% @spec node_name(k8s_endpoint) -> list()  
 %% @doc Return a full rabbit node name, appending hostname suffix
 %% @end
 %%
 node_name(Address) ->
   autocluster_util:node_name(
     autocluster_util:as_string(Address) ++ autocluster_config:get(k8s_hostname_suffix)).
 
+
+%% @spec maybe_ready_address(k8s_subsets()) -> list()
+%% @doc Return a list of ready nodes
+%% SubSet can contain also "notReadyAddresses"  
+%% @end
+%%
+maybe_ready_address(Subset) ->
+  case proplists:get_value(<<"addresses">>, Subset) of 
+      undefined -> autocluster_log:info("No nodes ready yet!"), [];
```

Comment:
Log message is not strictly true, we might have several subsets: some have ready addresses and others not. If we want to notify that there are not ready addresses, we could do something like:
```
maybe_ready_address(Subset) ->
case proplists:get_value(<<"notReadyAddresses">>, Subset) of
      undefined -> ok;
      _ -> autocluster_log:info("k8s endpoint listing returned nodes not yet ready", [])
  end,
  case proplists:get_value(<<"addresses">>, Subset) of
      undefined -> [];
      Address -> Address
  end.
```

@michaelklishin any thoughts?